roc/receiver.go

```diff
@@ -316,16 +341,23 @@ func (r *Receiver) SetReuseaddr(slot Slot, iface Interface, enabled bool) error
 // If endpoint has explicitly set zero port, the receiver is bound to a randomly
 // chosen ephemeral port. If the function succeeds, the actual port to which the
 // receiver was bound is written back to endpoint.
-func (r *Receiver) Bind(slot Slot, iface Interface, endpoint *Endpoint) error {
+func (r *Receiver) Bind(slot Slot, iface Interface, endpoint *Endpoint) (err error) {
+	logWrite(LogDebug,
+		"entering Receiver.Bind(): receiver=%p slot=%+v iface=%+v endpoint=%+v", r, slot, iface, endpoint,
+	)
+	defer logWrite(LogDebug, "leaving Receiver.Bind(): receiver=%p err=%v", r, err)
```

Comment:
Since Bind() modifies endpoint argument, let's log endpoint in defer statement as well.